meson.build

```diff
@@ -0,0 +1,92 @@
+project('elementary-tweaks', 'vala', 'c')
+
+switchboard_dep = dependency('switchboard-2.0')
+gettext_name = meson.project_name() + '-plug'
+pkgdatadir = join_paths(switchboard_dep.get_pkgconfig_variable('plugsdir'), 'personal')
+
+i18n = import('i18n')
+
+add_global_arguments(
+    '-DGETTEXT_PACKAGE="@0@"'.format(gettext_name),
+    language:'c'
+)
+
+subdir('po')
+
+conf_data = configuration_data()
+conf_data.set_quoted('PROJECT_NAME', meson.project_name())
+config_header = configure_file(
```

Comment:
This config is only ever used once, in `src/Tweaks.vala`. I suggest removing all config related code and just hardcode its 1 use in `Tweaks.vala`. This config is overkill otherwise.